tests/cli.rs

```diff
@@ -273,12 +273,21 @@ fn list_mutants_well_tested() {
 }
 
 #[test]
-fn list_mutants_well_tested_name_filter() {
+fn list_mutants_well_tested_examine_name_filter() {
```

Comment:
probably we should have a test for using them both together, too?

Reply:
excellent idea, I like it!

Reply:
I think I will add a new testree for this with having child directories.

Because the default behaviour of `cargo mutants` is to check the `src` directory, but I want a case where I `examine` a child directory but `exclude` a specific file.

```
cargo mutants --file src/module/utils/*.rs --exclude nested_function.rs
```